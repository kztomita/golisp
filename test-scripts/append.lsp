(print (append))
(print (append 'a))
(print (append '(a b)))
(print (append '(a b) '(c d)))
(print (append '(a b) '(c d) () nil '(e f)))
(print (append () '(a b)))
(print (append nil '(a b)))
(print (append '(a b) 'c))
(print (append () 'c))
;(print (append '(a b) 'c 'd))
