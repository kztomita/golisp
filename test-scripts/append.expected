
nil 
a 
(a b) 
(a b c d) 
(a b c d e f) 
(a b) 
(a b) 
(a b . c) 
c 
